### PR TITLE
BAH-2586 | Bahmni-Mart Application with Bahmni Docker

### PR DESCRIPTION
**Pull request by amankrayush**
The docker-compose.yaml file in the Bahmni Docker should be updated to include the Bahmni-Mart Application and the decided dependencies, so that every time we perform a docker-compose up along with the rest of the Bahmni Application the Bahmni-Mart should also be started.

## Jira
-----------
https://bahmni.atlassian.net/browse/BAH-2586

### CODE REVIEW

**Review by Ritzy-TWRKR — APPROVED** [review]

File: bahmni-proxy/resources/bahmni-proxy.conf [hunk 1]
```diff
@@ -56,6 +56,10 @@ ProxyPassReverse /oviyam2 http://dcm4chee:8055/oviyam2
 ProxyPass /atomfeed-console http://atomfeed-console:9080/atomfeed-console
 ProxyPassReverse /atomfeed-console http://atomfeed-console:9080/atomfeed-console
 
+#Metabase 
+ProxyPass /metabase http://metabase:3000
```
Comment (pajmera4) [inline bahmni-proxy/resources/bahmni-proxy.conf:60]: I see mb_site_url environment variable has not been added, so I hope this been tested properly

**Review by pajmera4 — CHANGES_REQUESTED** [review]
Also update readme for bahmni mart

**Review by pajmera4 — APPROVED** [review]

**Review by mohan-13 — APPROVED** [review]
Please update documentation [here](https://bahmni.atlassian.net/wiki/spaces/BAH/pages/3117482143/Metabase+Configuration+docker). Thnks

File: bahmni-proxy/resources/bahmni-proxy.conf (hunk 1 above)
Comment (amankrayush) [reply bahmni-proxy/resources/bahmni-proxy.conf:60]: Yes Priyanka

**Pull request merged**